README.md

```diff
@@ -17,10 +17,6 @@ Known Issues
 
 For samples using `Android.mk` build system with `ndk-build` see the [android-mk](https://github.com/googlesamples/android-ndk/tree/android-mk) branch.
 
-Debugging
----------
-- [hello-jni-codelab](https://codelabs.developers.google.com/codelabs/android-studio-jni/index.html?index=..%2F..%2Findex#0)
```

Comment:
thanks, it needed to be updated, may you replace it with this one:https://codelabs.developers.google.com/codelabs/android-studio-cmake/index.html?index=..%2F..%2Findex#0
if you could not access it, please let me know.